styles.css

```diff
@@ -0,0 +1,29 @@
+#header{
+    position: fixed;
+    top: 0;
+    min-height: 50px;
+    padding: 0px 10px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #eee;
+    
+    }
+    
+    body{
+    background-color: #eee;
+    font-family: 'Lato', sans-serif;}
+    .container {
+        max-width: 1000px;
+        width: 100%;
+        margin: 0 auto;
+      }
+      #hero {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+        height: 200px;
+        margin-top: 50px;
+      }
```

Comment:
- [x] You have done a very good job requesting this change, however, Please use kind words, be clear, and always give a clear explanation of why the requested change is necessary.

for example, you could say: :arrow_down: 

- [ ] Good job so far, :clap: , however, Please ensure that the above line of code is properly formatted to avoid throwing linter errors.  :slightly_smiling_face: